components/buttons.ts/ApplyJob.tsx

```diff
@@ -0,0 +1,28 @@
+import { Job } from '@/types/job';
```

Comment:
Why are the buttons component inside `button.ts` folder?

Reply:
I think you got a typo here in the folder name 😅

Reply:
That was a mistake it has already been resolved. 
Further work is still pending. I have made the Draft PR just to track the progress.